### PR TITLE
Enable `no_useless_else` rule in php-cs-fixer

### PR DESCRIPTION
**Pull request by rob006**
Continuation of #14100
See https://github.com/FriendsOfPHP/PHP-CS-Fixer

### CODE REVIEW

**Comment by samdark** [comment]
Merged. Thanks.

File: framework/i18n/Formatter.php [hunk 1]
```diff
@@ -826,28 +827,30 @@ public function asRelativeTime($value, $referenceTime = null)
             if ($interval->s == 0) {
                 return Yii::t('yii', 'just now', [], $this->locale);
             }
+
             return Yii::t('yii', 'in {delta, plural, =1{a second} other{# seconds}}', ['delta' => $interval->s], $this->locale);
-        } else {
```
Comment (cebe) [inline framework/i18n/Formatter.php]: in cases like this, else would be much easier to understand... :-(

**Comment by cebe** [comment]
In general I am fine with improving code style and also thanks for working on this @rob006 but at this scale its massivly breaking git blame, not sure its worth the pain... Maybe its enough for now?

File: framework/i18n/Formatter.php (hunk 1 above)
Comment (rob006) [inline framework/i18n/Formatter.php]: It would be easier to understand if we extract these blocks to separate methods.

**Comment by rob006** [comment]
@cebe I want to add few more fixers, but they should result less changes than this one. 

But if we want to force some code standards, we should do it now or never. The later we do this, the more changes it will require, give a greater impact on git blame and will be harder to merge changes between branches.
And I'm not sure if git blame is i worth of dropping automated CS fixing. Influence on git blame is temporary, with bad CS we will live forever :).

**Comment by samdark** [comment]
I'm for consistent code style.

**Pull request merged**